doc/releases/changelog-dev.md

```diff
@@ -365,6 +365,10 @@
   to a state and returns a new state.
   [(#3637)](https://github.com/PennyLaneAI/pennylane/pull/3637)
 
+* The `preprocess` function is added to `devices/qubit` that validates, expands, and transforms a batch
```

Comment:
>`added to devices/qubit`

What is qubit-specific about this function?

Reply:
The function is supposed to preprocess quantum scripts for the upcoming Python device, which is going to be a qubit state-vector simulator.